### ISSUE TITLE
Tag a new version

### ISSUE THREAD

**Issue by tristanbes** [tehplague/swiftmailer-mailgun-bundle#62]
Hello,

For Symfony4 users, only dev-master is compliant. Can you release a new version of the bundle please ?

**Comment by Nyholm** [comment]
Sure. Have you tested dev-master? Does it work for you?

**Comment by tristanbes** [comment]
Yes I just did test that: https://github.com/tehplague/swiftmailer-mailgun-bundle/issues/63 ;-)
